# independent-work
phylosim.py:
This paper describes the steps by which a new domain-level simulation of zinc fingers was developed. The simulation combines the use of multiple Python packages for phylogenetic tree creation and sequence evolution, as well as a unique representation of domain-level events (including duplications, losses, and speciations) in order to simulate domain evolution patterns as well as sequence level evolution of protein domains and the linker regions which surround them. The resulting orthogroups generated by the simulation were finally compared to actual zinc-finger orthologous groups in order to verify the simulationâ€™s accuracy. This simulation aims to provide new insight regarding the evolutionary mechanisms of zinc finger protein domain evolution and addresses a void in currently available domain-level simulation models.

orthosim.py:
This paper describes the steps by which a new phylogenetic tree reconstruction algorithm was developed. This algorithm directly builds upon previously conducted research in designing accurate simulations for protein domain evolution, and was developed using a combination of Python packages for creating and analyzing phylogenetic trees, widely available conventional tree reconstruction algorithms, a new method for reconstructing gene tree topologies from sequence groups. Multiple phylogeny reconstructions were generated from random, realistic domain-evolution simulations, and reconstructed trees were compared against simulated trees to evaluate reconstruction accuracy. This reconstruction algorithm attempts to provide novel insight regarding some techniques which may improve the reconstruction accuracy of widely available tree reconstruction algorithms.
